client/packages/invoices/src/InboundShipment/DetailView/modals/InboundServiceLineEdit/useDraftServiceLines.ts

```diff
@@ -46,12 +46,13 @@ export const useDraftServiceLines = () => {
   const update = (patch: RecordPatch<DraftInboundLine>) => {
     setDraftLines(currLines => {
       const newLines = currLines.map(line => {
-        const taxAmount =
-          (patch.totalBeforeTax ?? 0) * ((patch.taxPercentage ?? 0) / 100);
-        const totalAfterTax = (patch.totalBeforeTax ?? 0) + taxAmount;
-        const newPatch = { ...patch, totalAfterTax };
-        if (line.id === patch.id)
-          return { ...line, ...newPatch, isUpdated: true };
+        if (line.id === patch.id) {
```

Comment:
I can't see description of this change, what should it do, how to test it ?

Reply:
It came from [this suggestion](https://github.com/openmsupply/open-msupply/pull/1311#discussion_r1141466686), which wasn't taken up by the dev before merging. I still think that it's a good idea, and functionally equivalent. 

To test - check that the total value is updated in the inbound shipment service line edit modal, when you make a change to the tax or amount values. The PR has more details.. perhaps too many details!